lerobot/scripts/train.py

```diff
@@ -386,6 +390,16 @@ def evaluate_and_checkpoint_if_needed(step):
 
         step += 1
 
+    logging.info("End of offline training")
+
+    if cfg.training.online_steps == 0:
```

Comment:
As I've mentioned in the datset concat PR, ou can just drop this online training code as it is not reached, and actually ydoes nothing. I forgot to drop it as part of online training remove.

Reply:
I think this code logic will be used in the future. Let's merge it with this PR no?

Reply:
I don't feel too strongly about it. Unblocking.